src/com/mani/lang/core/Interpreter.java

```diff
@@ -391,7 +391,14 @@ public Object visitImportExpr(Expr.Import expr) {
                 return "Already loaded!";
             } else {
                 try {
-                    URL url = new URL("https://raw.githubusercontent.com/crazywolf132/Mani/master/stdlib/" + (String) result + ".mni");
+                    String stdlibURL = "https://raw.githubusercontent.com/crazywolf132/Mani/master/stdlib/";
```

Comment:
Let's move this to the Main file. That way we can create a function under the system API to change the location if we need.

Let's also use the new path. `https://raw.githubusercontent.com/Mani-Language/Mani/master/stdlib/`